.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
out folder is generated by Idea after running. It's also necessary to ignore

Reply:
Didn't notice it this time. Will do.

> out folder is generated by Idea after running. It's also necessary to ignore

Reply:
But "out" package is not visible in my directory.

> out folder is generated by Idea after running. It's also necessary to ignore

Reply:
Just run your main method, and it will be generated

Reply:
Actually, I have run it, please see the output below.
```
/Library/Java/JavaVirtualMachines/jdk-15.0.1.jdk/Contents/Home/bin/java -javaagent:/Applications/IntelliJ IDEA CE.app/Contents/lib/idea_rt.jar=59762:/Applications/IntelliJ IDEA CE.app/Contents/bin -Dfile.encoding=UTF-8 -classpath /Users/shake/Library/Caches/JetBrains/IdeaIC2020.3/compile-server/epam_java_automation_24211984/scratches/out Main
Hello world!

Process finished with exit code 0
```
But the `out/` folder was not generated.

Nevertheless I have added it to the `.gitignore` file in 75bf09a

Reply:
ok